docs/source/modin/supported/general_supported.rst

```diff
@@ -47,28 +47,21 @@ Data manipulations
-|                             |                                 | ``nrows``, ``keep_default_na``,  |                                                    |
-|                             |                                 | ``na_filter``, ``date_parser``,  | Supported parameters are ``sep``, ``header``,      |
-|                             |                                 | ``parse_dates``, ``verbose``,    | ``names``, ``index_col``, ``usecols``, ``dtype``,  |
-|                             |                                 | ``infer_datetime_format``,       | ``compression``, ``na_values``, ``escapechar``,    |
-|                             |                                 | ``date_format``,                 | ``skiprows``, ``encoding``, ``quotechar``, and     |
-|                             |                                 | ``keep_date_col``, ``dayfirst``, | ``skip_blanks_lines``.                             |
-|                             |                                 | ``cache_dates``, ``iterator``,   |                                                    |
-|                             |                                 | ``chunksize``, ``thousands``,    |                                                    |
-|                             |                                 | ``decimal``, ``lineterminator``, |                                                    |
-|                             |                                 | ``quoting``, ``doublequote``,    |                                                    |
-|                             |                                 | ``comment``, ``encoding_errors``,|                                                    |
-|                             |                                 | ``dialect``, ``on_bad_lines``,   |                                                    |
-|                             |                                 | ``low_memory``,                  |                                                    |
-|                             |                                 | ``delim_whitespace``,            |                                                    |
-|                             |                                 | ``memory_map``,                  |                                                    |
-|                             |                                 | ``float_precision``,             |                                                    |
-|                             |                                 | ``storage_options``,             |                                                    |
-|                             |                                 | ``dtype_backend``                |                                                    |
+| ``read_csv``                | P                               |                                  | Reads both local and staged file(s) into a Snowpark|
```

Comment:
The last column looks like it's missing a few words: "Note, the order of rows in the may differ from the order of rows in the original file(s)"